docs/runtime/go-metrics.md

```diff
@@ -0,0 +1,205 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Go Runtime
+--->
+
+# Semantic Conventions for Go Runtime Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+This document describes semantic conventions for Go runtime metrics in OpenTelemetry.
+These metrics are obtained from Go's [`runtime/metrics`][RuntimeMetrics] package.
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Go Memory](#go-memory)
+  - [Metric: `go.memory.used`](#metric-gomemoryused)
+  - [Metric: `go.memory.soft_limit`](#metric-gomemorysoft_limit)
+  - [Metric: `go.memory.allocated`](#metric-gomemoryallocated)
+  - [Metric: `go.memory.allocations`](#metric-gomemoryallocations)
+- [Go Garbage Collection](#go-garbage-collection)
+  - [Metric: `go.memory.gc.goal`](#metric-gomemorygcgoal)
+- [Go Goroutines](#go-goroutines)
+  - [Metric: `go.goroutine.count`](#metric-gogoroutinecount)
+- [Go Processor](#go-processor)
+  - [Metric: `go.processor.limit`](#metric-goprocessorlimit)
+- [Go Scheduler](#go-scheduler)
+  - [Metric: `go.schedule.duration`](#metric-goscheduleduration)
+- [Go Runtime Configuration](#go-runtime-configuration)
+  - [Metric: `go.config.gogc`](#metric-goconfiggogc)
+
+<!-- tocstop -->
+
+## Go Memory
+
+**Description:** Go runtime metrics captured under the namespace `go.memory.*`
+
+### Metric: `go.memory.used`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.go.memory.used(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `go.memory.used` | UpDownCounter | `By` | Memory used by the Go runtime. [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+**[1]:** Computed from `(/memory/classes/total:bytes - /memory/classes/heap/released:bytes)`.
+<!-- endsemconv -->
+
+<!-- semconv metric.go.memory.used(full) -->
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`go.memory.type`](/docs/attributes-registry/go.md) | string | The type of memory. | `other`; `stack` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+`go.memory.type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `stack` | Memory allocated from the heap that is reserved for stack space, whether or not it is currently in-use. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `other` | Memory used by the Go runtime, excluding other categories of memory usage. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+### Metric: `go.memory.soft_limit`
```

Comment:
I thought the conclusion from [this thread](https://github.com/open-telemetry/semantic-conventions/pull/981#discussion_r1593693033) was to go with `go.memory.limit`?

Reply:
yes, sorry.  Fixed.